packages/web-react/src/components/TextField/__tests__/TextField.test.tsx

```diff
@@ -18,6 +19,8 @@ describe('TextField', () => {
 
     validationStatePropsTest(TextField, 'TextField--');
 
+    validationTextPropsTest(TextField, '.TextField__message', type as TextFieldType);
```

Comment:
`TextField__validationText`? Or later?

Reply:
Later. We plan to do this with the v1 release as a BC.